easybuild/easyconfigs/p/protobuf/protobuf-3.6.1-fosscuda-2018b.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'protobuf'
+version = '3.6.1'
+
+homepage = 'https://github.com/google/protobuf/'
+description = """Google Protocol Buffers"""
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
```

Comment:
@akesandgren We already have a `protobuf-3.6.1-GCCcore-7.3.0.eb`, should be good enough, no?